prereqs

```diff
@@ -35,12 +48,51 @@ has() {
 }
 
 prereqs_check() {
-    local config_file=${1:-prereqs.conf}
     prereqs_found=true
-    set +e
-    # shellcheck source=/dev/null
-    . "${config_file}"
-    set -e
+    local config_file=${1}
+    if [[ -z ${config_file} ]]; then
+        # Get prereqs from README
```

Comment:
I think this should be a separate function (prereqs_check_readme, perhaps?) so that it can be tested.